test_palindrome.py

```diff
@@ -0,0 +1,197 @@
+from palindrome import is_palindrome
+
+
+def test_none_is_not_a_palindrome() -> None:
+    case, expectation = None, False
+    assert is_palindrome(case) == expectation, f"Expected '{expectation}' for '{case}'!"
```

Comment:
Tutaj myślę, że możesz użyć po prostu:

```
assert is_palindrome(case) is None
```

Zwracam uwagę na `is`, bo `None` jest w Pythonie singletonem, więc w zasadzie powinno się sprawdzać za pomocą `is`.

Reply:
W związku ze zmianą na obsługiwanie tylko stringów zmieniłem, że teraz jest rzucany ValueError w przypadku None.